addon/ng2/blueprints/component/index.js

```diff
@@ -4,7 +4,7 @@ var Blueprint = require('ember-cli/lib/models/blueprint');
 var dynamicPathParser = require('../../utilities/dynamic-path-parser');
 var addBarrelRegistration = require('../../utilities/barrel-management');
 var getFiles = Blueprint.prototype.files;
-const stringUtils = require('ember-cli/lib/utilities/string');
+const stringUtils = require('ember-cli-string-utils');
```

Comment:
What is the reason for using the separate `ember-cli-string-utils` instead of the built-in `ember-cli/lib/utilities/string`?